src/webview/js/js.js

```diff
@@ -81,6 +83,31 @@ if (!Element.prototype.closest) {
   };
 }
 
+/* Polyfill String#startsWith. Native in Mobile Safari 9, Chrome 49. */
+if (!String.prototype.startsWith) {
+  // $FlowFixMe (polyfill)
+  Object.defineProperty(String.prototype, 'startsWith', {
+    value(search, rawPos) {
+      const pos = rawPos > 0 ? rawPos | 0 : 0;
+      return this.substring(pos, pos + search.length) === search;
+    },
+  });
+}
+
+/* Polyfill String#includes. Native in Mobile Safari 9, Chrome 41. */
+if (!String.prototype.includes) {
+  // $FlowFixMe (polyfill)
+  String.prototype.includes = function includes(search, start = 0) {
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
```

Comment:
Hmm, why not just `indexOf` and test the return value?

At a quick test, I think this length-check case is inconsistent with the `includes` semantics:
```
$ node
> "abc".includes("", 100)
true
> "abc".indexOf("", 100)
3
```

Reply:
Yeah -- MDN is a public wiki, and it's a very good one but can still suffer from the weaknesses of the genre.

Basically I think of it a lot like the stronger subject areas of Wikipedia.

Also, this means you can Be Bold and fix it there for the next person :slightly_smiling_face:

Reply:
Also also, I find it helpful to have a comment like "Adapted from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes".

Reply:
There's so little of it left, I'm not sure that's true anymore! But it certainly holds for `startsWith`, so I'll add a note.

Reply:
(And the MDN update has been submitted.)